src/main/g8/src/main/scala/$package$/JsonFormats.scala

```diff
@@ -1,12 +1,12 @@
 package $package$
 
-import $package$.UserRegistryActor.ActionPerformed
+import $package$.UserRegistry.ActionPerformed
 
-//#json-support
-import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
+//#json-formats
 import spray.json.DefaultJsonProtocol
+import akka.http.scaladsl.marshallers.sprayjson.SprayJsonSupport
 
-trait JsonSupport extends SprayJsonSupport {
+object JsonFormats extends SprayJsonSupport {
```

Comment:
Let's also use an import for `SprayJsonSupport` directly in the routes. It's not needed here and if people have another object with json implicits they might otherwise accidentally `extends SprayJsonSupport` another time and then the implicits break down in the routes.